src/services/iasql.ts

```diff
@@ -179,16 +180,31 @@ export async function runSql(dbAlias: string, uid: string, sql: string) {
     connMain = await createConnection({ ...dbMan.baseConnConfig, database, name: pass, });
     await connMain.query(dbMan.newPostgresRoleQuery(user, pass, database));
     await connMain.query(dbMan.grantPostgresRoleQuery(user));
-    connTemp = await createConnection({ ...dbMan.baseConnConfig, database, name: user, });
+    connTemp = new pg.Client({
```

Comment:
I think it is possible to get the bare pool/connection from `pg` from the createconnection like we do here https://github.com/iasql/iasql-engine/blob/main/src/services/scheduler.ts#L29.  It would be ideal so we consistently use the same config, but not blocking.

Reply:
So this *has* to be a separate connection (see the old code being replaced) so I didn't think it was that big of a deal to cut out the middleman here, too.

Reply:
I understand it it is a separate connection. I meant consistency in configuring and getting a connection since we do it in a few places across the codebase. That was the purpose of the db manager at least? The TypeORM bug is unfortunate. Like this is totally fine though. Mostly a suggestion

Reply:
To be clear, I was also worried about Typeorm still returning a "tampered" object here, so I went with the clearest way to do this change. :)